### DISCUSSION TITLE
something went wrong bug

### DISCUSSION THREAD

**Discussion by beingjoey** [community/community#37422] [Other Feature Feedback, Questions, & Ideas]
I get "Sorry something went wrong" message all the time in github with chrome. And I can not do anything because of that error. With edge there is no this error. 
I used to clean the brower data, cookie, but it did not work.

**Comment by radar-blip** [comment]
What are you trying to do when you get his error?

**Comment by beingjoey** [comment]
The Recent Repositories, Recent activity, Lastest changes list are all empty. And I can not comment this issues, right now. I have to use edge.

**Comment by beingjoey** [comment]
This problem has persisted for weeks，in my different pc with the same google id in chrome。

**Reply by radar-blip** [reply]
You can try filing a bug report here: [Bug-Report](https://support.github.com/contact/bug-report) (let them know what browser you are using).

It could also be account related, if thats the case, try here: [Support page](https://support.github.com/contact/account)

No guarantees but worth a try.

**Reply by beingjoey** [reply]
thanks for your reply

**Reply by radar-blip** [reply]
No problem, hope you get it figured out. Seems something strange is happening there.
